app/services/tplink_service.py

```diff
@@ -68,6 +70,21 @@ def get_power_data_month(self, device_name=None, device_filter=None):
 
         return self._jsonify(usage)
 
+    def get_device_sys_info(self, device_id):
+        device_data = self._device_manager.get_devices()
```

Comment:
looks like I need to add a "get device by ID" capability to the API library - it'll basically do the same thing (get all then filter) but then you wouldn't have to

I think the only specific-device lookup now gets by a device name

Reply:
I agree that this is the case. I went through the exercise of exploring the API library when implementing this code.